README.md

```diff
@@ -1,3 +1,55 @@
 # jab
 
```

Comment:
možda napisat da jab znači JSON Api Bridge

Reply:
aha to znaci ok